sharepoint_connector_config.yml

```diff
@@ -0,0 +1,42 @@
+#Configurations for the SharePoint 2016 Connector
+
+#SharePoint 2016 configuration settings
+#The client id generated while app registration on sharepoint server to access the contents
+sharepoint.client_id: dummy
+#The secret generated while app registration on sharepoint server
+sharepoint.client_secret: dummy
+#The authentication realm of a SharePoint farm
+sharepoint.realm: dummy
+#The address of the sharepoint farm. Example: http://sharepoint-host:14293/
+sharepoint.host_url: http://10.0.9.211:14682
+#Specifies the site collections whose contents the user wants to fetch and index.
+sharepoint.site_collections: 
+    - Connector
+#Elastic WorkPlace configuration settings
```

Comment:
Just FYI, it should be referred to as `Workplace Search`. Also note that _Workplace Search_ is a solution within the _Enterprise Search_ product.